install/apache-guacamole-install.sh

```diff
@@ -122,6 +127,7 @@ Restart=always
 WantedBy=multi-user.target
 EOF
 systemctl -q enable --now tomcat guacd mysql
+systemctl start guacd
```

Comment:
```suggestion
```
Should not be needed. The Service gets enabled and strated in the line above.

Reply:
done

Reply:
In my previous tests, the `guacd` service didn’t start after being enabled. This is why I explicitly added the command to start the service during the installation process. Without this, users were required to manually start the service or restart their LXC container, as it didn’t seem to work immediately after installation.
root@apache-guacamole:~# systemctl status guacd.service 
○ guacd.service - LSB: Guacamole proxy daemon
     Loaded: loaded (/etc/init.d/guacd; generated)
     Active: inactive (dead)
       Docs: man:systemd-sysv-generator(8)